kyo-core/shared/src/main/scala/kyo/ios.scala

```diff
@@ -59,7 +59,7 @@ sealed trait IOs extends Effect[IOs]:
         end try
     end attempt
 
-    def handle[T, S, U >: T, S2](v: => T < S)(
+    def catching[T, S, U >: T, S2](v: => T < S)(
```

Comment:
Renamed to be more consistent with `Aborts.catching`. I think `handle` can also give an impression that the effect itself is getting handled.